editors/code/src/config.ts

```diff
@@ -224,8 +224,8 @@ export class Config {
         return this.get<string | undefined>("cargoRunner");
     }
 
-    get runnableEnv() {
-        const item = this.get<any>("runnableEnv");
+    get runnablesExtraEnv() {
+        const item = this.get<any>("runnables.extraEnv");
```

Comment:
We should also try parsing `runnableEnv` for a bit longer so we don't immediately break peoples configs.